chainer/links/connection/gru.py

```diff
@@ -100,16 +113,30 @@ class StatefulGRU(GRUBase):
     Args:
         in_size(int): Dimension of input vector :math:`x`.
         out_size(int): Dimension of hidden vector :math:`h`.
+        init: A callable that takes a tuple of the matrix shape and returns a
```

Comment:
> What should I put as the type since it can be one of many things? (callable or matrix or scalar)?

I imagined the list of possible types of arguments as you suggest.